packages/dashboard/src/lib/components/CreateEntity.svelte

```diff
@@ -0,0 +1,67 @@
+<script>
+	import { Button, Row } from 'carbon-components-svelte';
+	import { Search } from 'carbon-components-svelte';
+	import ArrowRight16 from 'carbon-icons-svelte/lib/ArrowRIght16';
+	import ArrowLeft16 from 'carbon-icons-svelte/lib/ArrowLeft16';
+	import Number_1 from 'carbon-icons-svelte/lib/Number_132';
+	import { Pagination } from 'carbon-components-svelte';
+
+	import { Tag } from 'carbon-components-svelte';
+
+	export let title = '';
+	export let description = '';
+	export let name = '';
+
+	// TODO
+	function goToCreatePage() {
+		// go to create an organisation or project page
+		if (name == 'organization') {
+			// go to create organisation page
+		} else if (name == 'project') {
+			// go to create project page
+		}
+	}
+	function showMore() {
+		// show more information about the selected organisation or project
+	}
+	function prev() {
+		// go to prev slide of organisations
+	}
+	function next() {
+		// go to next slide of organisations
+	}
+</script>
+
+<div style="padding: 2rem;">
+	<h1>{title}</h1>
+	<p>{description}</p>
+</div>
+
+<!-- TODO: implement search through the list of projects/organisations -->
+<div class="flex flex-row gap-64 justify-between" style="padding: 2rem;">
+	<Button kind="tertiary" on:click={goToCreatePage}>Create {name}</Button>
+	<Search placeholder="Search {name}" />
+</div>
+
+<!-- TODO: the projects/organisations should be read from a data file -->
+<div
+	class="flex flex-row justify-between"
+	style="padding-left: 2rem; padding-right: 2rem;padding-bottom: 2rem;"
```

Comment:
You can add unlimited tailwind utility classes to `class`. Therefore, the `style` attribute is redundant. Instead do the following

```HTML
<div class="flex flex-row justify-between pl-2 pr-2 pb-2" >
```

[Here](https://tailwindcss.com/docs/padding) you can find all padding classes from tailwind.